src/System.Net.HttpListener/src/System/Net/Windows/WebSockets/WebSocketBase.cs

```diff
@@ -1210,7 +1210,7 @@ internal void ThrowIfClosedOrAborted()
             if (State == WebSocketState.Closed || State == WebSocketState.Aborted)
             {
                 throw new WebSocketException(WebSocketError.InvalidState,
-                    SR.Format(SR.net_WebSockets_InvalidState_ClosedOrAborted, GetType().FullName, State));
+                    SR.Format(SR.net_WebSockets_InvalidState_ClosedOrAborted, GetType().ToString(), State));
```

Comment:
We had a discussion with .NET Native folks on this.  Looking in the earlier, outdated comments on this PR:

>GetType().FullName will throw a MissingMetadataException if the type has reflection metadata optimized away. GetType().ToString() will always work: it will either be the type name, or a string that has the address of a type if the type doesn't have reflection metadata. For cases like this, ToString() is definitely the preferred pattern (if nobody else wants to machine-parse the type name later).